test-files/cmd/glacial/RndCropRandomCmdTest/out.osm

```diff
@@ -555,9 +555,9 @@
         <nd ref="-1026"/>
         <nd ref="-1027"/>
         <nd ref="-170959"/>
-        <tag k="name" v="K St NW"/>
         <tag k="error:circular" v="5"/>
         <tag k="highway" v="secondary"/>
+        <tag k="name" v="K St NW"/>
```

Comment:
Tag ordering is the only thing in this file because the test wasn't using `-C Testing.conf`, it is now.